src/oph/ehoks/hoks/handler.clj

```diff
@@ -323,9 +328,12 @@
         :summary "Luo uuden HOKSin"
         :body [hoks hoks-schema/HOKSLuonti]
         :return (rest/response schema/POSTResponse :id s/Int)
-        (check-oids-match hoks)
-        (add-oppija-to-index hoks)
-        (add-opiskeluoikeus-to-index hoks)
+        (let [opiskeluoikeudet (oppijaindex/fetch-opiskeluoikeudet-by-oppija-id
+                                 (:oppija-oid hoks))]
+          (check-opiskeluoikeus-match hoks opiskeluoikeudet)
+          (add-oppija-to-index hoks)
+          (add-opiskeluoikeus-to-index hoks)
+          (add-hankintakoulutukset hoks opiskeluoikeudet))
```

Comment:
Nämä voisi ehkä olla samalla tavalla nimetty. Että joko kaikki to-index lisällä tai ilman.

Reply:
Totta, lisätty -to-index hankintakoulutuksiin.